edgelet/iotedge/src/check/checks/container_resolve_parent_hostname.rs

```diff
@@ -12,7 +12,7 @@ impl Checker for ContainerResolveParentHostname {
         "container-resolve-parent-hostname"
     }
     fn description(&self) -> &'static str {
-        "config.yaml parent hostname is resolvable from inside container"
+        "config.toml parent hostname is resolvable from inside container"
```

Comment:
`configuration`

Reply:
Fixed

Reply:
Actually, this shouldn't even have anything before `parent`

Reply:
Fixed